### PR TITLE
Basic readthedocs site

### PR DESCRIPTION
**Pull request by PrajwalPrashanth**
- Added custom css to match the color scheme of jvn.io website
- Sekelton of side nav bar( things to be documented)
- Added autodoc(generate from docstrings), linkcode(link to source), napoleon( to support google style docstring) sphinx extensions

Can view at - https://prajwalprashanth-demo.readthedocs.io/en/prajwalprashanth-ch193-set-up-deploy-basic-readthedocs-site/

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -9,12 +9,11 @@
 # If extensions (or modules to document with autodoc) are in another directory,
```
Comment (aakashns) [inline docs/conf.py]: Please remove the helper comments. You can keep the title e.g. 'Project Information' or 'General Configuration', but remove the descriptions. If required, you can add a link to the original conf file with the details documentation at the top of the file.

File: docs/conf.py [hunk 2]
```diff
@@ -48,11 +48,35 @@
...
 html_static_path = ['_static']
+html_style = 'css/custom.css'
+
+html_logo = 'jovian_favicon.png'
+html_favicon = 'jovian_favicon.png'
+html_icon = 'jovian_favicon.png'
+
+
+def setup(app):
+    # app setup hook
```
Comment (aakashns) [inline docs/conf.py]: Use docstrings

File: jovian/__init__.py [hunk 3]
```diff
@@ -19,23 +19,14 @@
...
-    """Save the notebook, capture environment and upload to the cloud for sharing.
+    """Commits a Jupyter Notebook with its environment to Jovian.
 
-    In most cases, commit works well with the default arguments. It attempts to 
-    1. Save the Jupyter notebook
-    2. Upload the notebook (and additional scripts, CSVs etc.) to https://jvn.io 
-    3. Capture the python environment (using Anaconda or pip)
-    4. Upload the python environment to cloud
+    Saves the checkpoint of the notebook, capture the required dependencies from the python environment and uploads the notebook, env file, additional files like scripts, csv etc. to https://www.jvn.io . Capturing the python environment ensures that the notebook can be reproduced and 
+    executed easily using the ***{links to reprodue notebooks}.***
```
Comment (aakashns) [inline jovian/__init__.py]: typo in `reprodue`

Comment (aakashns) [reply jovian/__init__.py]: Pls. spellcheck all the text before merging.

File: docs/conf.py [hunk 4]
```diff
@@ -48,11 +48,35 @@
...
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-
+# html_theme_options = {'logo_only': True}
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+html_style = 'css/custom.css'
+
+html_logo = 'jovian_favicon.png'
```
Comment (aakashns) [inline docs/conf.py]: you can try using the full logo here.

https://i.imgur.com/K3kjlCm.png

File: jovian/__init__.py (hunk 3 above)
Comment (PrajwalPrashanth) [reply jovian/__init__.py]: Did the spellcheck for all files and resolved the typos

**Pull request merged**